### PR TITLE
Rendering Payment page in Webview

### PR DESCRIPTION
**Pull request by imsaravana369**
In order to support the UPI Intent feature provided by Juspay for flutter webview merchants.
hyper_webview_flutter

### CODE REVIEW

File: hyper_webview_flutter/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# hyper_webview_flutter
+
```
Comment (Subhash703) [inline hyper_webview_flutter/README.md:2]: @imsaravana369 replace it with one which is there in hyper-sdk-flutter plugin

File: hyper_webview_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```
Comment (Subhash703) [inline hyper_webview_flutter/CHANGELOG.md:3]: modify this

File: hyper_webview_flutter/LICENSE [hunk 3]
```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```
Comment (Subhash703) [inline hyper_webview_flutter/LICENSE:1]: @imsaravana369  add the license from hyper-sdk-flutter plugin

**Comment by imsaravana369** [comment]
Created a separate repo for this plugin.
PR here https://github.com/juspay/hyper-webview-flutter/pull/1

**Pull request closed without merging**